src/unstract/sdk/adapters/utils.py

```diff
@@ -25,17 +28,29 @@ def get_msg_from_request_exc(
         Returns:
             str: Error message returned by the server
         """
-        if hasattr(err, "response"):
-            err_response: Response = err.response  # type: ignore
-            if err_response.headers["Content-Type"] == "application/json":
-                err_json = err_response.json()
-                if message_key in err_json:
-                    return str(err_json[message_key])
-            elif err_response.headers["Content-Type"] == "text/plain":
-                return err_response.text  # type: ignore
+        if not hasattr(err, "response"):
+            return default_err
+
+        err_response: Response = err.response  # type: ignore
+        err_content_type = err_response.headers["Content-Type"]
```

Comment:
Will there be a case when "Content-Type" attribute is absent and should we handle that, if yes?